torchvision/prototype/transforms/_utils.py

```diff
@@ -45,12 +45,18 @@ def query_chw(sample: Any) -> Tuple[int, int, int]:
     return chws.pop()
 
 
+def _isinstance(obj: Any, types_or_checks: Tuple[Union[Type, Callable[[Any], bool]], ...]) -> bool:
```

Comment:
Could you also use this in `has_all`?

Reply:
I think we can change easily for-loops types -> objects into objects -> types.

Reply:
True, I didn't see that. Could you add a comment to `has_all` why we are not using it, since that is not obvious?